client/src/components/Metrics/MetricsEpoch/MetricsEpochGridBelowThreshold/MetricsEpochGridBelowThreshold.tsx

```diff
@@ -34,12 +35,9 @@ const MetricsEpochGridBelowThreshold: FC<MetricsEpochGridBelowThresholdProps> =
     },
   }));
   const { data: cryptoValues, error } = useCryptoValues(displayCurrency);
-  const { data: projectsDonors } = useProjectsDonors(
-    isDecisionWindowOpen && epoch === lastEpoch ? undefined : epoch,
-  );
 
   const projectsBelowThreshold =
-    Object.keys(projectsDonors).length -
+    (projectsEpoch?.projectsAddresses.length || 0) -
```

Comment:
What is the reason for this change?

Reply:
`projectsDonors` return only projects that have received donation (here we need all projects).